spec/rspec/mocks/verifying_doubles/class_double_with_class_loaded_spec.rb

```diff
@@ -165,8 +165,6 @@ class << self
 
         context "on a class with a private `new`" do
           it 'uses the method signature from `#initialize` for arg verification' do
-            pending "Failing on JRuby due to https://github.com/jruby/jruby/issues/2565" if RSpec::Support::Ruby.jruby?
```

Comment:
jruby 9.2.13.0 can pass this so removing it, otherwise RSpec is complaining this is actually passing.